entity-service/src/test/kotlin/com/egm/stellio/entity/web/EntityHandlerTests.kt

```diff
@@ -561,6 +678,63 @@ class EntityHandlerTests {
             )
     }
 
+    @Test
+    fun `get entity by id should include relationship temporal properties if query param sysAttrs is present`() {
```

Comment:
It looks quite similar to `get entity by id should correctly serialize properties of type DateTime and display sysAttrs asked`, no?

Reply:
OK I am changing it